### ISSUE TITLE
package-> artifact attribute in a component

### ISSUE THREAD

**Issue by kostjazvankovich** [serverless/components#305]
# Description
Is it possible to set the following attribute for a lambda function in a component: **package-> artifact**? 

I use this feature for a Java runtime.
## Additional Data

E.g. stack trace, version, operating system, ...

**Comment by brianneisler** [comment]
@kostjazvankovich yep, that would definitely be possible. What would you expect data shape to look like and can you give me an example of what it would look like to use that in a variable?

**Comment by kostjazvankovich** [comment]
> @kostjazvankovich yep, that would definitely be possible. What would you expect data shape to look like and can you give me an example of what it would look like to use that in a variable?

Hello Brian! 

I have a **Clojure** handler for a lambda function. For the right aws deployment of lambda function I should specify the following attributes in my serverless config:

service: wac-operator

**package**:
  **artifact**: target/wac-0.1.0-SNAPSHOT-standalone.jar

provider:
  name: aws
  **runtime: java8**

functions:
  handle:
    handler: **brand.wac.Handler**
    events:
      - http: post

runtime is java8 and package is specified with standalone jar file.

with serverless components I don't know how it is possible to specify **package artifact** and java **runtime**.  When I specify runtime java8 my lambda function is created with nodeJs runtime.

Best regard,
Kostya

**Comment by kostjazvankovich** [comment]
Hi @brianneisler ,

in was-lambda component **zipFile** property could be configurable:

`code: {
      zipFile: target/wac-0.1.0-SNAPSHOT-standalone.jar
    }`

Best regard,
Kostya

**Comment by kostjazvankovich** [comment]
Hi @brianneisler, could I starting investigating and fixing of the issue?
